packages/dd-trace/test/appsec/reporter.spec.js

```diff
@@ -60,6 +60,7 @@ describe('reporter', () => {
         'user-agent',
         'x-forwarded-for',
         'x-client-ip'
+
```

Comment:
uh ?

Reply:
:racehorse: :racehorse: :racehorse: you are fast!